docs/fixers.rst

```diff
@@ -21,6 +21,12 @@ then use the ``--no-six`` flag.
 Fixers use the API defined by 2to3. For details of how this works, and how to
 implement your own fixers, see `Extending 2to3 with your own fixers, at
 python3porting.com <http://python3porting.com/fixers.html>`_.
+``python-modernize`` will try to load fixers whose full dotted-path is specified
+as a ``-f`` argument, but will fail if they are not found. Note that, by
```

Comment:
Something I picked up from time spent editing Wikipedia: the words "Note that" can almost always be deleted with no change to the meaning. And indeed that works here!